### PR TITLE
update dashboard

### PR DESCRIPTION
**Pull request by YitongFeng**
1. change active GPUs gauge's threshold
2. change color schema
3. change the units of disk and network to bps
4.  modify the metrics
5. Change gpu page vertical view into horizontal:
image

### CODE REVIEW

**Comment by fanyangCS** [comment]
Please change the unit of network traffic and disk of nodeview to Mbps or Gbps also. http://10.151.40.179:3000/dashboard/db/pai_nodeview?refresh=30s&orgId=1

File: service-deployment/src/grafana/pai-clusterview-dashboard.json.template [hunk 1]
```diff
@@ -579,7 +579,7 @@
           },
           "yaxes": [
             {
-              "format": "bytes",
+              "format": "bps",
```
Comment (sterowang) [inline service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: How about using "Bps"? It looks more familiar than "bps".

Comment (hwuu) [reply service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: Agree. 'b' usually refers to 'bit', 'B' is 'Byte'.

Comment (sterowang) [reply service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: If use "Bps", remember to remove "* 8" then.

Comment (YitongFeng) [reply service-deployment/src/grafana/pai-clusterview-dashboard.json.template]: OK, changed to Bps. @hwuu @sterowang

**Comment by YitongFeng** [comment]
I can't reply under your comment so I write here. @fanyangCS  I have modified but havn't deployed at the big cluster.

**Pull request merged**